packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -722,6 +724,7 @@ export abstract class SharedObject<
 	protected abstract summarizeCore(
 		serializer: IFluidSerializer,
 		telemetryContext?: ITelemetryContext,
+		incrementalContext?: IIncrementalContext,
```

Comment:
Please open a bug to make it non-optional in _next_. Likely same for telemetryContext.

Reply:
I don't think this is simple. When we call getGCData, we also call summarizeCore which does not have the incrementalSummaryContext. Here's the task: AB#3881